src/ToricVarieties/LineBundles/constructors.jl

```diff
@@ -4,7 +4,7 @@
 
 abstract type ToricCoherentSheaf end
 
-struct ToricLineBundle <: ToricCoherentSheaf
+@attributes mutable struct ToricLineBundle <: ToricCoherentSheaf
     variety::AbstractNormalToricVariety
     divisor_class::GrpAbFinGenElem
```

Comment:
```suggestion
    divisor_class::GrpAbFinGenElem
    
    ToricLineBundle(variety::AbstractNormalToricVariety, divisor_class::GrpAbFinGenElem) = new(variety, divisor_class)
```
Then you don't need the `Dict()` below.

Reply:
I think I see this now. Thank you for elaborating. I will try this out later and then update this PR.